classes/handlers/handler.ts

```
@@ -24,16 +24,23 @@ class Handler {
     return members;
   }
 
-  async summonPokemon(message: any, mode = 'normal'): Promise<void> {
+  async summonPokemon(
+    message: any,
+    mode = 'normal',
```

ESLint

[error] Default parameters should be last.

(default-param-last)